### PR TITLE
Fix tests for input views

### PR DESCRIPTION
**Pull request by latentflip**
We should merge this asap (imo) but the commented AmpersandInputView lines in the test file are because if https://github.com/AmpersandJS/ampersand-input-view/issues/26

### CODE REVIEW

**Comment by bear** [comment]
@latentflip any reason to not merge this?

File: ampersand-view-conventions.js [hunk 1]
```diff
@@ -1,6 +1,31 @@
...
+
+    //defined on the object, but not truthy
+    if (object.hasOwnProperty(fieldName)) { return true; }
+
+    //Defined as a getter/setter on the prototype chain
+    var current = object.constructor.prototype;
+    var fieldDefinition;
+
+    while (current !== Object.getPrototypeOf({})) {
+        fieldDefinition = Object.getOwnPropertyDescriptor(current, 'value');
```
Comment (cdaringe) [inline ampersand-view-conventions.js]: this is probably supposed to be `fieldName`?

**Comment by cdaringe** [comment]
**bump**.  reviving this in effort to support https://github.com/AmpersandJS/ampersand-form-view/issues/28

**Comment by cdaringe** [comment]
additionally, may we simultaneously reduce the stringency of [this line](https://github.com/AmpersandJS/ampersand-view-conventions/pull/5/files#diff-15331390f4c8c1e49f6771cd4205b6acR105) to perhaps just a `t.ok`?  [This is still inwork](https://github.com/AmpersandJS/ampersand-input-view/issues/56), thus input view tests will still break

**Comment by wraithgar** [comment]
@latentflip :shipit:

**Comment by wraithgar** [comment]
published as `v1.1.5`

**Pull request merged**